pkg/controller/apiclient/mem/binding.go

```diff
@@ -37,6 +38,11 @@ func newBindingClient() *bindingClient {
 	return &bindingClient{bindings: make(map[string]*servicecatalog.Binding)}
 }
 
+// NewPopulatedBindingClient returns a pre-populated in-memory binding client implementation
+func NewPopulatedBindingClient(bin map[string]*servicecatalog.Binding) apiclient.BindingClient {
```

Comment:
Another non-blocking nit: I would consider `bindings` more readable than `bin`. My objection isn't to the shortening of the word, but rather to the fact that `bin` sounds singular.